stac_fastapi/api/stac_fastapi/api/app.py

```diff
@@ -7,32 +7,36 @@
 from fastapi import APIRouter, FastAPI
 from fastapi.openapi.utils import get_openapi
 from fastapi.params import Depends
+from pydantic import BaseModel
 from stac_pydantic import Catalog, Collection, Item, ItemCollection
 from stac_pydantic.api import ConformanceClasses, LandingPage
 from stac_pydantic.api.collections import Collections
 from stac_pydantic.version import STAC_VERSION
 from starlette.responses import JSONResponse, Response
 
-from stac_fastapi.api.errors import DEFAULT_STATUS_CODES, add_exception_handlers
+from stac_fastapi.api.errors import (DEFAULT_STATUS_CODES,
```

Comment:
these imports don't look like they were formatted using black. Can you make sure you've got the formatted installed in your dev environment.

Reply:
Reformatted with black